lib/stoplight/notifier/bugsnag.rb

```diff
@@ -0,0 +1,37 @@
+# coding: utf-8
+
+module Stoplight
+  module Notifier
+    # @see Base
+    class Bugsnag < Base
+      DEFAULT_OPTIONS = {
+        severity: 'info'
+      }
+
+      StoplightStatusChange = Class.new(StandardError)
+
+      # @return [Proc]
+      attr_reader :formatter
+      # @return [::Bugsnag]
+      attr_reader :bugsnag
+      # @return [Hash{Symbol => Object}]
+      attr_reader :options
+
+      # @param bugsnag [::Bugsnag]
+      # @param formatter [Proc, nil]
+      # @param options [Hash{Symbol => Object}]
+      # @option options [String] :severity
+      def initialize(bugsnag, formatter = nil, options = {})
+        @bugsnag = bugsnag
+        @formatter = formatter || Default::FORMATTER
+        @options = DEFAULT_OPTIONS.merge(options)
+      end
+
+      def notify(light, from_color, to_color, error)
+        formatter.call(light, from_color, to_color, error).tap do |message|
```

Comment:
I would prefer to not use `tap`. I think [what the HipChat notifier does](https://github.com/orgsync/stoplight/blob/d195eaadb37a085ff79f7bbd594010cf606b8feb/lib/stoplight/notifier/hip_chat.rb#L37-L39) is fine.